.github/workflows/cypress-tests-runner.yml

```diff
@@ -22,6 +22,45 @@ env:
   RUST_MIN_STACK: 10485760
 
 jobs:
+  formatter:
```

Comment:
This job should only run if we're running against a PR (we should not be modifying anything in case of merge queue). Can add an `if` attribute on the entire job.